subprojects/wrapper/src/main/java/org/gradle/wrapper/GradleWrapperMain.java

```diff
@@ -66,9 +66,10 @@ public static void main(String[] args) throws Exception {
                 new BootstrapMainStarter());
     }
 
-    private static void addSystemProperties(File gradleHome, File rootDir) {
-        System.getProperties().putAll(SystemPropertiesHandler.getSystemProperties(new File(gradleHome, "gradle.properties")));
```

Comment:
I found the code clearer before, i.e. explicitly calling System.getProperties to make it obvious what we're changing here.

Reply:
To me, "systemProperties" makes just as clear as "System.getProperties()" that we are dealing with system properties here.

Reply:
Thanks! Sorry for the late reply (I was out for a bit longer). I'll see that we can get this merged now.